File: client/src/app/dataModels/common/contextResponse.ts
export interface ContextResponse {
    isAuthenticated: boolean;
    lang: string;
    userId?: string;
    firstName?: string;
    lastName?: string;
    roleName?: string;
    refreshTokenExpirationDateTime?: string;
    assignmentId?: string;
}

export interface CurrentAssignmentInfo {
    id?: string;
}

File: server/src/ApplicationCore/Handlers/Context/ContextResponse.cs
namespace ApplicationCore.Handlers.Context;

public class ContextResponse
{
    public bool IsAuthenticated { get; set; }

    public string Lang { get; set; }

    public Guid? UserId { get; set; }

    public string FirstName { get; set; }

    public string LastName { get; set; }

    public string RoleName { get; set; }

    public DateTime? RefreshTokenExpirationDateTime { get; set; }
}